deploy/helm/magda/values.yaml

```diff
@@ -13,6 +13,8 @@ global:
   logLevel: INFO
   noDbAuth: false
   namespace: default
+  gapiId: 
+    - UA-92539508-3
```

Comment:
This should probably be `[]` by default then we can add this and the other id in https://github.com/magda-io/magda-config-data.gov.au :)